### PR TITLE
42 post on facebook

### PR DESCRIPTION
**Pull request by oshtohryn**
## TRELLO

* [Main Trello ticket](https://trello.com/b/BJDdC443/mytrip)

## Code reviewers

- [ ] @Ashiato
- [ ] @black-art
- [ ] @hrytskivr
- [ ] @KarnaushenkoVO
- [ ] @PanLeon
- [ ] @spycheast
- [ ] @yuriiZherebukh

### Second Level Review

- [ ] @byshovets
- [ ] @lhalam

### CODE REVIEW

File: myTrip/registration/views.py [hunk 1]
```diff
@@ -20,7 +23,7 @@
                       "client_id={client_id}&redirect_uri={redirect_uri}&"
                       "client_secret={client_secret}&code={code}")
 FACEBOOK_USER_URL = 'https://graph.facebook.com/me?access_token={token}'
-FACEBOOK_REDIRECT_URL = 'http://triptrck.com/api/v1/auth/facebook_login/'
+FACEBOOK_REDIRECT_URL = 'http://triptrck.com:8000/api/v1/auth/facebook_login/'
```
Comment (oshtohryn) [inline myTrip/registration/views.py]: import `HOST ` from settings

File: myTrip/registration/views.py [hunk 2]
```diff
@@ -33,7 +36,7 @@
         {url}
 """
 
-RESTORE_PASS_URL = ("http://localhost:8000/restore-password/{token}/")
+RESTORE_PASS_URL = ("http://triptrck.com:8000/restore-password/{token}/")
```
Comment (oshtohryn) [inline myTrip/registration/views.py]: remuve ()

File: myTrip/registration/views.py [hunk 3]
```diff
@@ -112,6 +120,8 @@ def logout(request):
             auth.logout(request)
             response = response_200_logout_successful
             response.delete_cookie('user_id')
+            response.delete_cookie('facebook-token')
+            response.delete_cookie('userid')
```
Comment (oshtohryn) [inline myTrip/registration/views.py]: remuve  response.delete_cookie('userid')

File: myTrip/static/src/facebooksdk.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+<div id="fb-root"></div>
+<script>(function(d, s, id) {
+  var js, fjs = d.getElementsByTagName(s)[0];
```
Comment (oshtohryn) [inline myTrip/static/src/facebooksdk.js]: const

File: myTrip/static/src/facebooksdk.js [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+  if (d.getElementById(id)) return;
+  js = d.createElement(s); js.id = id;
+  js.src = "//connect.facebook.net/en_US/sdk.js#xfbml=1&version=v2.10&appId=1317135221766606";
```
Comment (oshtohryn) [inline myTrip/static/src/facebooksdk.js]: import .env (js) appId

File: myTrip/utils/facebookPost.py [hunk 6]
```diff
@@ -0,0 +1,36 @@
+"""Module handle post's on facebook."""
+import ast
+import facebook
+import requests
+from django.http import HttpResponse
+
+from trip.models import Trip
+
+
+def postOnFacebook(request, trip_id):
```
Comment (oshtohryn) [inline myTrip/utils/facebookPost.py]: remuve

**Review by lhalam — APPROVED** [review]

**Pull request merged**